README.md

```diff
@@ -54,50 +54,47 @@ Project is designed to utilize django's powerful features.
+2. Development is done on the `dev_new` branch. Ensure you are working either on `dev_new` or a branch that has branched off of `dev_new`. To switch to a branch, use `git checkout [branch_name]`.
+3. Install pipenv with pip using `pip install pipenv`.
+   - If pip is not recognized, you can do `py -m ensurepip` to check that pip is installed, then restart your Terminal.
+4. **IMPORTANT** Get the .env file from another contributor and store it on your root directory '/SITE' if you haven't already before running this command.
+5. Initialize the virtual environment and install virtual environment dependencies using `pipenv install`.
+6. Activate the virtual environment using `pipenv shell`.
+7. Go to `Site/meshapp/` and run an `npm install`. You can change your directory to this folder using `cd meshapp`.
+8. Build the frontend by running `npm run build`.
+9. Return to the root directory and start the backend server / run the application with `python manage.py runserver`.
+    - `cd ..` (Windows) or `cd -` (MacOS) to go back a directory.
+
+**MYSQL SETUP** (If you are on a Mac, consider switching to Windows if you have one)
+1. Download MySQL over [here](https://dev.mysql.com/downloads/installer/). Download the larger file.
+2. Do the setup and do the full download (spam next until you see the password portion).
+3. Make a password for the root account or a custom username.
+   - Make sure you remember your passwords and username (username is by default root).
+4. Write the username and password in your .env file.
+5. Spam next until you finish and then you’re done.
+
+Next we're onto Django.
+1. Open a terminal and make sure MySQL is installed by typing in `mysql -V`
+   - If the command doesn’t work it may be because mysql isn’t added to your path. The default location for mysql server should be: C:\Program Files\MySQL\MySQL Server 8.0\bin\mysql.exe. A possible solution can be found [here](https://stackoverflow.com/questions/5920136/mysql-is-not-recognised-as-an-internal-or-external-command-operable-program-or-b).
+   - If it says Access Denied, move onto the next step.
+2. Login to MySQL by using  `mysql --u [username] --p` and then type in the password you saved previously when prompted.
+   - FYI: You should manually type the password as pasting it may not work.
+4. You should be prompted into the MySQL shell, where you will then create a database by using `CREATE DATABASE mesh;`.
+5. Confirm your database is created with `SHOW DATABASES;`.
+6. Make sure your pip environment is opened with `pipenv shell`.
+7. Type in `python manage.py makemigrations`.
+    - You will most likely be given warnings. Warnings are fine errors aren’t. Google your errors.
+8. Type in `python manage.py migrate`.
+    - Similar to the previous step warnings are fine, errors aren’t.
+9. Make sure your frontend is set up with `npm run build`, then run `python manage.py runserver`, and you should be good to go.
 
 #### Tips:
```

Comment:
Do you want to add the scripts from https://github.com/LetsMesh/Site/blob/PlainOlSoapBar-patch-1/package.json  ?

Reply:
Sure, what command do you use to run these scripts?

Reply:
Should be  npm run [command]